### PR TITLE
Move to jdk8 and grails 2.5.5

### PR DESCRIPTION
**Pull request by charvolant**
Version shifted to 1.9.0-SNAPSHOT to recognise changes in JDK/grails version
Updated to jdk8 and grails 2.5.5
Minor bug fixes from testing
Unit tests work
Integration tests work

### CODE REVIEW

**Comment by charvolant** [comment]
Upgrade to grails 2.5.6 and JDK 8
Upgrade to ala-bootstrap3
Resources framework as-is

File: grails-app/controllers/au/org/ala/collectory/LicenceController.groovy [hunk 1]
```diff
@@ -55,7 +55,7 @@ class LicenceController {
             return
         }
 
-        [licenceInstance: licenceInstance]
+        [licenceInsnce: licenceInstance]
```
Comment (djtfmartin) [inline grails-app/controllers/au/org/ala/collectory/LicenceController.groovy]: is this abbreviation intentional ?

**Comment by djtfmartin** [comment]
This looks good to me. Ive reviewed the code. Mammoth update - thank you !

File: grails-app/controllers/au/org/ala/collectory/LicenceController.groovy (hunk 1 above)
Comment (charvolant) [reply grails-app/controllers/au/org/ala/collectory/LicenceController.groovy]: No. I'll check to see whether it's mysteriously covered by other things.

**Comment by charvolant** [comment]
Corrected licenceInsnce error (and added a few images that were supposed to be included)

**Pull request merged**